CHANGELOG.md

```diff
@@ -15,15 +15,31 @@ Features, Bug Fixes, Breaking Changes, Deprecated
 
 ## Unreleased
 
+### API breaking
+
+- We have updated the default behaviour of algob deployer for loading data from checkpoint to be queried by  "app/lsig" name. The existing functions have been changed to `<func>ByFile` (legacy functions based on file querying):
+
+  - Application:
+    * Exisiting `getApp(approval.py, clear.py)` has been changed to `getAppByFile(approval.py, clear.py)`.
+    * New `getApp(appName)` function queries app info using the app name.
+
+  - Smart signatures:
+    * Exisiting `getDelegatedLsig(lsig.py)`, `getContractLsig(lsig.py)` has been changed to `getDelegatedLsigByFile(lsig.py)`, `getContractLsigByFile(lsig.py)` respectively.
+    * New `getApp(appName)` function queries app info using the app name.
+    * Existing `fundLsig(lsig.py, ..)` function has been changed to `fundLsigByFile(lsig.py, ..)`. Now `fundLsig(lsigName, ..)` will take lsig name.
+    * Existing `mkDelegatedLsig(fileName, signer, ..)`, `mkContractLsig(fileName, ..)` has been changed to `mkDelegatedLsigByFile(fileName, signer, ..)`, `mkContractLsigByFile(fileName, ..)` respectively.
+    * New functions take the lsigName as a required paramter:
+      - `mkDelegatedLsig(fileName, lsigName, signer)`
+      - `mkContractLsig(fileName, lsigName)`.
```

Comment:
please explain what's `lsigName` and how it is used.

Reply:
ok